src/shaders/_prelude_shadow.fragment.glsl

```diff
@@ -1,25 +1,37 @@
 #ifdef RENDER_SHADOWS
 
-uniform sampler2D u_shadowmap_0;
-uniform sampler2D u_shadowmap_1;
+uniform highp sampler2D u_shadowmap_0;
```

Comment:
Maybe we need to enclose this on `#ifdef DEPTH_TEXTURE`, as precisely we are going to hurt more in the case the device doesn't support Depth textures, which should be the worst case now. (As discussed offline previous samplers are just 8 bit, so we don't need highp). 

What do you think?

Reply:
I can do that 👍